packages/ckeditor5-core/src/editor/editorui.js

```diff
@@ -53,6 +52,14 @@ export default class EditorUI {
 		 */
 		this.focusTracker = new FocusTracker();
 
+		/**
+		 * Stores viewport offsets from every direction.
+		 *
+		 * @readonly
```

Comment:
This will not appear in the docs AFAIK. You need to use `@member` if that's set by `this.set()`. JSDoc is able to recognize `this.x = 1`, but not this.

Reply:
Also, you need to define the type of this property and please provide some example on how to override it.
As for the type, you can go the easy way and define it as `Object` and explain what keys it has in the text.